juju_verify/utils/unit.py

```diff
@@ -37,8 +37,12 @@ def get_cache_key(unit: Unit, action: str, **params: Any) -> int:
     )
 
 
-async def run_action(unit: Unit, action: str, use_cache: bool = True,
-                     params: Optional[Dict[str, Any]] = None) -> Action:
+async def run_action(
+    unit: Unit,
+    action: str,
+    use_cache: bool = True,
+    params: Optional[Dict[str, Any]] = None,
+) -> Action:
```

Comment:
Maybe it's just me but I think that these func signatures looks worse after `black`